apps/asn1pars.c

```diff
@@ -136,7 +136,7 @@ int asn1parse_main(int argc, char **argv)
             genconf = opt_arg();
             break;
         case OPT_STRICTPEM:
-            strictpem = 1;
+            /* accepted for compatibility reasons */
```

Comment:
how about "... for backwards compatibility */" instead ?

Reply:
If you prefer that, I can change it. However, I prefer [backward compatibilty](https://www.merriam-webster.com/dictionary/backward%20compatibility) (without the 's') which according to Merriam-Webster (and Google's hit count) is slightly more common.

Reply:
I think "backward compatibility" provides clearer context—the code is as is to work with scripts that use old calling convention/parameters, not because it will change in future